haml/_header.html.haml

```diff
@@ -26,14 +26,12 @@
                   = link["title"]
               - else
                 %a.cads-header__hyperlink{ href: link["url"] }= link["title"]
-
-          - if sign_in_out
```

Comment:
This should be reverted as not all users of the design-system will supply a sign_in_out link. As just one example, not all micro-sites need it.